push/views.py

```diff
@@ -1,7 +1,8 @@
 from django.contrib import messages
+from django.contrib.auth.mixins import LoginRequiredMixin, UserPassesTestMixin
 from django.shortcuts import get_object_or_404, redirect
+from django.utils.translation import ugettext as _
```

Comment:
Can `ugettext_lazy` be used here, too? Or is the non-lazy function needed in Views for some reason?